XML/TestCases/PerformanceTests.xml

```diff
@@ -124,7 +124,7 @@
         <Category>Performance</Category>
         <Area>NETWORK</Area>
         <Tags>network,hv_netvsc</Tags>
-        <Priority></Priority>
+        <Priority>2</Priority>
```

Comment:
I'm not sure If it's good to set 2. This test case always fails for no enough resources. @LiliDeng How do you think?